x/programs/rust/wasmlanche-sdk/src/program.rs

```diff
@@ -33,6 +33,12 @@ impl BorshDeserialize for DeferDeserialize {
     }
 }
 
+impl Into<Vec<u8>> for DeferDeserialize {
```

Comment:
The `Into` was deliberately implemented because we only want the `Vec<u8>` -> `DeferDeserialize` to happen with the help of `borsh`.

Reply:
This doesn't make sense. You should be implementing `From<DeferDeserialize>`

Reply:
irrelevant because it has been removed.